File: src/2022/day02/src/main/java/ca/terrylockett/aoc2022/day02/Day02Runner.java
package ca.terrylockett.aoc2022.day02;

import ca.terrylockett.aoccommon.inputfilefinder.Resources;

public class Day02Runner {

	public static void main(String[] args) throws Exception {

		String filePath = Resources.getInputFilePath("input.txt").orElseThrow();

		// ******** part 01 **********
		System.out.println("Part 01 Game Total: " + RockPaperScissorsGame.playGames(filePath));

		// ******** part 02 **********
		System.out.println("Part 02 Game Total: " + RockPaperScissorsGame.playGames02(filePath));

	}
}

File: src/2023/day02/src/main/kotlin/ca/terrylockett/aoc2023/day02/Day02Runner.kt
package ca.terrylockett.aoc2023.day02

import ca.terrylockett.aoccommon.inputfilefinder.Resources

fun main() {
	val inputFile: String = Resources.getInputFilePath("input.txt").orElseThrow()

	println("2023 day02 part1: ${CubeGame.playGamePart1(inputFile)}")
	println("2023 day02 part2: ${CubeGame.playGamePart2(inputFile)}")
}
